plugins/ptp_operator/metrics/metrics.go

```diff
@@ -150,6 +153,14 @@ func (p *PTPEventManager) ExtractMetrics(msg string) {
 		p.ParseTBCLogs(processName, configName, output, fields, ptpStats)
 	case syncE4lProcessName:
 		p.ParseSyncELogs(processName, configName, output, fields, ptpStats)
+	case chronydProcessName:
+		ptpStats.CheckSource(ClockRealTime, configName, processName)
+		ptpStats[ClockRealTime].SetProcessName(processName)
+		if chronydValidSource.MatchString(output) {
+			p.GenPTPEvent(profileName, ptpStats[ClockRealTime], ClockRealTime, int64(0), ptp.LOCKED, ptp.OsClockSyncStateChange)
```

Comment:
should ptp metrics be updated too?

Reply:
where would that be updated?

Reply:
similar to process is running or not. 
Also in linuxptpdaemon when you stopping phc2sys process,  I think it  will update process down in the log, but then you should do similar to chronyD (steop|start)